### DISCUSSION TITLE
Welcome to astra-assistants-api Discussions!

### DISCUSSION THREAD

**Discussion by phact** [datastax/astra-assistants-api#41] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Rishabh-14** [comment]
how to integrate deepseek_coder-v2
as it is mentioned in client that it is supported https://github.com/datastax/astra-assistants-api/tree/main/client
# DeepSeek models
export DEEPSEEK_API_KEY=""
